mozci/console/commands/regression.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from cleo.commands.command import Command
+from cleo.helpers import argument
+
+from mozci.push import Push
+
+
+class RegressionCommand(Command):
```

Comment:
We could re-use ClassifyCommand, maybe add an option to limit to builds only so it's faster to iterate, and add an option to just visualize results